test/testcafe/spec/AdminConsentView_spec.js

```diff
@@ -5,7 +5,7 @@ import SuccessPageObject from '../framework/page-objects/SuccessPageObject';
 
 import xhrConsentAdmin from '../../../playground/mocks/data/idp/idx/consent-admin';
 import xhrSuccess from '../../../playground/mocks/data/idp/idx/success';
-
+// import { a11yCheck } from '../framework/shared'; // OKTA-TODO
```

Comment:
Will create follow-up JIRAs for home teams to address. Most are straightforward and do not require UX input.